Sources/ProjectDescription/TargetAction.swift

```diff
@@ -45,6 +45,9 @@ public struct TargetAction: Codable, Equatable {
     /// Whether to skip running this script in incremental builds, if nothing has changed
     public let basedOnDependencyAnalysis: Bool?
 
+    /// Whether this script only runs on install builds (default is false)
```

Comment:
```suggestion
    /// Whether this action only runs on install builds (default is false)
```

Users might use the initializer with `tool` and not `script`, so this might feel a little bit misleading (although, factually correct). I'd search&replace it in other occasions as well

Reply:
done